### PR TITLE
Fix source install on CentOS-6

### PR DESCRIPTION
**Pull request by shoekstra**
### Description

Currently compiling from source fails on CentOS 6, this PR fixes that.

### Contribution Check List

- [ ] All tests pass.
- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by damacus** [comment]
Are you still using centos-6?

Were really thinking of dropping all non-systemd platforms, ie centos-6 and amazon Linux 1

**Comment by shoekstra** [comment]
Alas, yes, we still have some in production as it's EOL in 2020.

Been using the cookbook fine so far but as it fails when installing from source - and I have a fix - I thought I would contribute upstream.

File: resources/install.rb [hunk 1]
```diff
@@ -79,6 +79,10 @@
     make_cmd = "make TARGET=#{new_resource.source_target_os}"
     make_cmd << " CPU=#{new_resource.source_target_cpu}" unless new_resource.source_target_cpu.nil?
     make_cmd << " ARCH=#{new_resource.source_target_arch}" unless new_resource.source_target_arch.nil?
+    if node['platform_family'] == 'rhel' && node['platform_version'].to_i < 7
+      make_cmd << ' USE_RT=1'
+      make_cmd << ' USE_CRYPT_H=1'
+    end
```
Comment (rshade) [inline resources/install.rb]: I know I am being pedantic, but could this be moved to the end, before extra_config?

Comment (JohnRoesler) [reply resources/install.rb]: I just pushed that change up

**Pull request merged**